libkbfs/data_types.go

```diff
@@ -58,6 +58,7 @@ type TeamInfo struct {
 	TID          keybase1.TeamID
 	CryptKeys    map[KeyGen]kbfscrypto.TLFCryptKey
 	LatestKeyGen KeyGen
+	RootID       keybase1.TeamID // for subteams only
```

Comment:
I don't think I use it that way anywhere, but it should reliably work that way if we want.  Though I think it's more clear to do `TID.IsSubTeam()`.